core-ui/src/components/Extensibility/components/Widget.js

```diff
@@ -158,6 +164,7 @@ export function Widget({ structure, value, inlineRenderer, ...props }) {
         Renderer={Renderer}
         value={item}
         structure={structure}
+        inlineContext={inlineContext}
```

Comment:
it should be enough to pass this through using `props`

Reply:
I decide to go for "visible props" from the next comments.

Reply:
but in this component it's not used for anything, so it's just unnecessary extra code